### PR TITLE
KRV-4562 : Enhance the scripts to pick the namespace from the manifests

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
Enhance the scripts to pick the namespace from the manifests

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/254 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
** Installation with olm : ** 
install_olm

** Installation of olm without namespace available on system : ** 
image

** Uninstall with olm : ** 
uninstall_olm

### CODE REVIEW

File: scripts/install_olm.sh [hunk 1]
```diff
@@ -75,6 +74,11 @@ function check_for_olm_components() {
 function check_existing_installation() {
   log separator
   echo "Checking for existing installation of Dell CSM Operator"
+  # get namespace from YAML file for deployment
+  NS_STRING=$(cat ${DEPLOYDIR}/operator_community.yaml | grep "namespace:" | head -1)
```
Comment (rensyct) [inline scripts/install_olm.sh:78]: use cat ${MANIFEST_FILE} instead of hardcoding

**Review by medegw01 — DISMISSED** [review]

File: scripts/install_olm.sh [hunk 2]
```diff
@@ -75,6 +74,11 @@ function check_for_olm_components() {
 function check_existing_installation() {
   log separator
   echo "Checking for existing installation of Dell CSM Operator"
+  # get namespace from YAML file for deployment
+  NS_STRING=$(cat ${MANIFEST_FILE} | grep "namespace:" | head -1)
+  # find the namespace from the filtered string
```
Comment (rensyct) [inline scripts/install_olm.sh:79]: please add the null check for NS_STRING

File: scripts/install_olm.sh (hunk 1 above)
Comment (karthikk92) [reply scripts/install_olm.sh]: tc

File: scripts/install_olm.sh (hunk 2 above)
Comment (karthikk92) [reply scripts/install_olm.sh:83]: tc

**Review by rensyct — APPROVED** [review]

**Review by panigs7 — APPROVED** [review]

**Pull request merged**